docs/architecture.md

```diff
@@ -239,3 +239,28 @@ NetworkPolicy implementation.
 
 As described earlier, Antrea Controller leverages the Kubernetes apiserver
 library to build the API and communication channel to Agents.
+
+### IPSec encryption
```

Comment:
Pretty minor point, but it seems all the documentation online - including OVS documentation - spells it IPsec and not IPSec.

Reply:
Ok. It is the first time I know this. I can change documentation to use IPsec, but probably not the code. You think what we should do for the yaml file?

Reply:
starting with the documentation sounds good